src/components/course-administration/reset-password.cjsx

```diff
@@ -0,0 +1,44 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+{RosterActions} = require '../../flux/roster'
+
+PasswordResetField = React.createClass
+  displayName: 'ResetStudentPasswordField'
+  mixins: [React.addons.LinkedStateMixin]
+
+  getInitialState: ->
+    password: ''
+
+  getValue: -> @state.password
+
+  componentDidMount: ->
+    React.findDOMNode(@refs.input).focus()
```

Comment:
This focus is performed on a plain `<input />` component, while `add-student.cjsx` operates on a `TutorInput`  While it would be possible for `AddStudent` to reach down into `TutorInput`'s dom and muck around, it seemed better to ask the `TutorInput` to focus itself.